modules/auxiliary/analyze/pcap_ot_iot_analyzer.rb

```diff
@@ -0,0 +1,218 @@
+      '5800' => 'VNC - Virtual Network Computing',
+      '6379' => 'Redis Database',
+      '7400' => 'CODESYS - Industrial Control System Protocol',
+      '7401' => 'CODESYS - Industrial Control System Protocol',
+      '7402' => 'CODESYS - Industrial Control System Protocol',
+      '7626' => 'SIMATICS7 - Siemens Industrial Control System',
+      '8000' => 'QNX QCONN',
+      '8080' => 'HTTP Alternate',
+      '8443' => 'HTTPS Alternate',
+      '8888' => 'HuaweiSymantec',
+      '9100' => 'PDL Data Stream - Printer Job Language',
+      '9200' => 'Elasticsearch',
+      '9600' => 'OMRON FINS - Industrial Control System Protocol',
+      '11112' => 'DICOM - Digital Imaging and Communications in Medicine',
+      '18245' => 'GE SRTP - GE Intelligent Platforms SRTP',
+      '18246' => 'GE SRTP - GE Intelligent Platforms SRTP',
+      '20000' => 'DNP3 - Distributed Network Protocol',
+      '27017' => 'MongoDB Database',
+      '34962' => 'PROFInet RT Unicast',
+      '34963' => 'PROFInet RT Multicast',
+      '34964' => 'PROFInet Context Manager',
+      '34980' => 'EtherCAT - Industrial Ethernet Protocol',
+      '41100' => 'Siemens SICAM',
+      '44818' => 'EtherNet/IP - Industrial Ethernet Protocol',
+      '47808' => 'BACnet - Building Automation and Control Networks',
+      '48898' => 'ADS - Automation Device Specification',
+      '50000' => 'Siemens S7 Scalance',
+      '55000' => 'FL-net - Industrial Control System Protocol',
+      '56001' => 'Guardian AST',
+      '62900' => 'ABB Ranger 2003',
+      '1024' => 'Wago 750',
+      '1102' => 'Wago 750',
+      '5050' => 'Carel Industrial Control System',
+      '4910' => 'Niagara Fox Industrial Control System',
+      '4843' => 'OPC UA - Open Platform Communications Unified Architecture',
+      '9999' => 'Telnet Alternative'
+    }
+
+    # Read custom ports from FILE_PORT
+    custom_ports = {}
+    File.readlines(datastore['FILE_PORT']).each do |line|
+      port, protocol = line.strip.split('/')
+      custom_ports[port] = protocol if port && protocol
+    end
+
+    # Initialize PCAP reader
+    begin
+      pcap = PacketFu::PcapFile.read_packets(datastore['FILE_PCAP'])
+    rescue StandardError => e
+      print_error("Error reading PCAP file: #{e.message}")
+      print_error("File details:")
+      print_error("  Path: #{datastore['FILE_PCAP']}")
+      print_error("  Size: #{File.size(datastore['FILE_PCAP'])} bytes")
+      print_error("  Magic bytes: #{File.open(datastore['FILE_PCAP'], 'rb') { |f| f.read(4).unpack('H*')[0] }}")
+      return
+    end
+
+    # Open output CSV file
+    CSV.open(datastore['FILE_OUT'], 'w') do |csv|
+      csv << ['Date/Time', 'Source IP', 'Destination IP', 'Port', 'Protocol', 'MAC Address', 'Packet Size', 'OT/IOT Description']
+
+      pcap.each do |packet|
+        begin
+          pkt = PacketFu::Packet.parse(packet)
+          next unless pkt.is_ip?
+
+          timestamp = Time.at(packet.timestamp.to_f).strftime('%y/%m/%d - %H:%M:%S')
```

Comment:
Can we switch this to the ISO8601 format please? it's YYYY-MM-DDTHH:MM:SS IIRC.
```suggestion
          timestamp = Time.at(packet.timestamp.to_f).strftime()
```